rotkehlchen/chain/evm/proxies_inquirer.py

```diff
@@ -144,15 +133,27 @@ def get_or_query_summer_fi_proxy(
         """Return summer.fi proxies if they exist for the list of addresses.
         Finds proxy addresses by inspecting the decoded history events.
         """
+        return self._get_proxy_addresses_from_history(
```

Comment:
This `get_or_query_summer_fi_proxy` ... is it used anywhere? I dont' see it in a quick look at my checkout. And if yes can't you just replace it with a call to `_get_proxy_addresses_from_history` directly?